fix(ses): work around #2348 linenumber bug (#2355)
endojs/endo: packages/ses/NEWS.md

User-visible changes in SES:

# Next release

> - When running Ava tests on Node written in TypeScript, the SES error taming

Author: There's no need to mention Ava here. This should be for all TypeScript on Node on v8.

Author: Done

- Adds support for module descriptors better aligned with XS.

> - When running Ava tests on Node written in TypeScript, the SES error taming
>   gives line-numbers into the generated JavaScript, in which all compiled code
>   is on line 1. This happens even with the normal development-time lockdown
>   options setting

Reviewer: ```suggestion
- When running transpiled code on Node, the SES error taming
  gives line-numbers into the generated JavaScript, which often don't match the 
  the original lines. This happens even with the normal development-time lockdown
  options setting,
```

Author: Done, thanks.

  ```js
  errorTaming: 'unsafe'
  ```
  or setting the environment variable
  ```sh
  $ export LOCKDOWN_ERROR_TAMING=unsafe
  ```

>   In addition to the normal `'safe'` and `'unsafe'` settings, this release

Reviewer: ```suggestion
  To get the original line numbers, this release
```

Author: Done

  adds `'unsafe-debug'`. This `errorTaming: 'unsafe-debug'` setting
  should be used ***during development only*** when you can
  sacrifice more security for a better debugging experience, as explained at
  [`errorTaming` Options](https://github.com/endojs/endo/blob/master/packages/ses/docs/lockdown.md#errortaming-options).

>   With this setting, when running Ava tests written in TypeScript on Node,

Reviewer: ```suggestion
  With this setting, when running transpiled code on Node (e.g. tests written in TypeScript),
```

Author: Done

>   the stacktrace line-numbers should point back into the original
>   TypeScript source, as they would on Node without SES.

Reviewer: ```suggestion
  the stacktrace line-numbers point back into the original
  source, as they do on Node without SES.
```

Author: Done. Together, all this reads much better. Thanks!
